noticeboardapp/templates/noticeboard/delete_all_notices.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'homepageapp/base.html' %}
+
+{% block content %}
+  <h4 style="color:red;">Do you really want to delete all notices?</h4>
+
+    <form method="post">{% csrf_token %}
+    <input type="submit" value="Confirm" class="btn btn-info">
```

Comment:
Please move to django forms: https://docs.djangoproject.com/en/4.2/topics/forms/